script/sync_prod_to_gcp/webnode_pdf_request.py

```diff
@@ -125,6 +128,27 @@ def ping_callback(message: Message) -> None:
             message.ack()
             return
 
+        message_age: timedelta = datetime.utcnow() - message.publish_time
+        compilation_timeout = int(os.environ.get("TEX_COMPILATION_TIMEOUT_MINUTES", "30"))
+        if message_age > timedelta(minutes=compilation_timeout):
+            help_needed = os.environ.get("TEX_COMPILATION_RECIPIENT", "help@arxiv.org")
+            subject = f"TeX compilation for {paper_id}v{version} failed"
+            mail_body = f"Hello EUTA,\nTex compilation for {paper_id}v{version} has failed. Please resolve the issue.\n\nThis message is generated by a bot on arxiv-sync.serverfarm.cornell.edu.\n"
+            cmd = ["/usr/bin/mail", "-r", "developers@arxiv.org", "-s", subject, help_needed]
```

Comment:
Will that work in the gcloud in the future?

Reply:
This is for a systemd service running on sync-node at CIT so this is tailored for it. The resulting PDF is checked on the CIT's file server so it's not possible to run on GCP.

Reply:
More over, once genpdf becomes the solo compiler, this will go away.